gcp/main.tf

```diff
@@ -19,68 +19,11 @@ provider "google" {
 # This code is compatible with Terraform 4.25.0 and versions that are backwards compatible to 4.25.0.
 # For information about validating this Terraform code, see https://developer.hashicorp.com/terraform/tutorials/gcp-get-started/google-cloud-platform-build#format-and-validate-the-configuration
 
-# EC instance
-resource "google_compute_instance" "instance-20240922-191308" {
-  boot_disk {
-    auto_delete = true
-    device_name = "instance-20240922-191308"
+module "pplox-web-deployment" {
+  source = "../modules/gcp-compute-engine"
 
-    initialize_params {
-      image = "projects/ubuntu-os-cloud/global/images/ubuntu-2404-noble-amd64-v20240911"
-      size  = 10
-      type  = "pd-balanced"
-    }
-
-    mode = "READ_WRITE"
-  }
-
-  can_ip_forward      = false
-  deletion_protection = false
-  enable_display      = false
-
-  labels = {
-    goog-ec-src           = "vm_add-tf"
-    goog-ops-agent-policy = "v2-x86-template-1-3-0"
-  }
-
-  machine_type = "e2-micro"
-
-  metadata = {
-    enable-osconfig = "TRUE"
-  }
-
-  name = "instance-20240922-191308"
-
-  network_interface {
-    access_config {
-      network_tier = "PREMIUM"
-    }
-
-    queue_count = 0
-    stack_type  = "IPV4_ONLY"
-    subnetwork  = "projects/poised-bot-273002/regions/us-central1/subnetworks/default"
-  }
-
-  scheduling {
-    automatic_restart   = true
-    on_host_maintenance = "MIGRATE"
-    preemptible         = false
-    provisioning_model  = "STANDARD"
-  }
-
-  service_account {
```

Comment:
The service account that was auto-generated by GCE's `Equivalent Code` is not carried over to the new instance. I have to read up on what this is used for but for now the new module can function without it.